app/extensions/string_extensions.rb

```diff
@@ -517,6 +517,45 @@ def gsub_html_special_chars
       html_safe # rubocop:disable Rails/OutputSafety
   end
 
+  # For integration test comparisons:
+  # Render special encoded characters as they appear in HTML
+  def render_html
+    CGI.unescapeHTML(self)
+  end
+
+  # Insert a line break between the scientific name and the author
+  # (for styling taxonomic names legibly)
+  def break_name
```

Comment:
Did you intend for these changes to go out with this PR?  I would, in fact, like to see what this looks like, but it seems to be entirely unrelated to CRUDifying the account and admin controllers.

Totally unimportant.  Let's just say the purpose of this comment is to bring these changes to the attention of other potential revewiers! :)

Reply:
Right. These are for the Observations show view, namings table, where the namings are kind of hard to read on a phone. Breaking the name and putting the author in slightly smaller text made them a lot easier to scan. 

I also would like to use these two methods in the handsome "matrix box" update i've been sitting on for nearly 3 years, but that is still in traffic waiting behind controller refactors and Bootstrap 4.